Detectors/FIT/FT0/calibration/testWorkflow/GlobalOffsetsCalibrationSpec.h

```diff
@@ -9,38 +9,44 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
```

Comment:
In general, `testWorkflow` directory contains mostly production code (and only w few test-examples), so perhaps it should be renamed to just `workflow`?